doc/man3/SSL_write.pod

```diff
@@ -102,18 +108,22 @@ Old documentation indicated a difference between 0 and -1, and that -1 was
 retryable.
 You should instead call SSL_get_error() to find out if it's retryable.
 
+For SSL_sendfile(), the return value is the return code of the sendfile system
+call.
```

Comment:
sendfile() returns a ssize_t, this functions returns an int

Reply:
It seems it can also return -1 for other reasons. sendfile() says to check errno in that case, which isn't going to be helpful.

Reply:
Thanks for the feedback, I've reworked the error reporting a little to be less Linux specific, while updating the documentation to be closer to SSL_write.